cli/src/torii/mod.rs

```diff
@@ -137,7 +142,7 @@ impl Error {
         let mut idx = 0_i32;
         let mut err_opt = Some(err);
         while let Some(e) = err_opt {
-            s += &format!("    {}: {}\n", idx, &e.to_string());
+            write!(s, "    {idx}: {}", &e.to_string()).expect("Valid");
```

Comment:
it doesn't ring true to me that this write would necessarily always succeed

Reply:
In each case in which it would reasonably fail (OOM, non-UTF characters in Display), panicking is sensible. Also, unlike the previous code, this is explicitly causing an error.